saleor/graphql/utils/__init__.py

```diff
@@ -51,6 +48,12 @@
     QueryCostError,
 ]
 
+AVAILABLE_SOURCE_SERVICE_NAMES_FOR_SPAN_TAG = {
+    "saleor.dashboard",
+    "saleor.dashboard.plg",
```

Comment:
It's a minor thing, but why not use `playground` instead of `plg` to make it more readable?

Reply:
I will change it :)